docs/columns.md

```diff
@@ -189,4 +191,40 @@ You can assign any [HTML Event](https://www.w3schools.com/tags/ref_eventattribut
 ```
 
 ## <a name='headerEvents'>column.headerEvents - [Object]</a>
-`headerEvents` same as [`column.events`](#events) but this is for header column.
\ No newline at end of file
+`headerEvents` same as [`column.events`](#events) but this is for header column.
+
+## <a name='attrs'>column.attrs - [Object | Function]</a>
+Via `attrs` property, You can costomize table column [HTML attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes) which allow user to configure the elements or adjust their behavior. It takes `Object` and `callback function` is also acceptable.
+
+```js
+{
+  // omit...
+  attrs: (cell, row, colIndex) => ({
+    // return customized HTML attribute here
+  })
+}
+```
+
+#### * Caution
+
+If `column.classes`, `column.style`, `column.title`, `column.hidden` or `column.align` was given at the same time, property `attrs` has lower priorty and it will be overwrited.
```

Comment:
Hi @AllenFang:
Thanks for your reminder. I've add corresponding test for this PR. 

However, I didn't mention `column.events` because it's not in the list of [HTML attribute reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes). Although `column.attrs.onClick` is still work-able but bad practice. I thought users should put their events in the `column.events` instead.

Reply:
sure, it's workable but not recommand